packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,66 @@
+While Lit allows rendering values straight out of an iterable, it would be nice
+to have the ability to apply some transformation like adding markup for each
+item. Lit provides a built-in `map` directive that works on any iterable such as
```

Comment:
```suggestion
Lit supports rendering iterables of items directly in templates, but you often need to render a _template_ that uses the items, rather than the items directly. Lit provides a built-in `map` directive that lets you transform the items in an iterable, usually to create a template from them. The map directive works with any iterable such as...
```